frontend/src/js/components/releases/dialogs/AddArtifact.tsx

```diff
@@ -67,29 +90,48 @@ const shortenFileName = name => {
   return name;
 };
 
-export const ArtifactUpload = ({ setSnackbar, updateCreation }) => {
+const singleFileLimit = 536870912; //512MiB
+const menderFileLimit = 10737418240; //10GiB
+const reFilename = new RegExp(/^[a-z0-9.,_-]+$/i);
```

Comment:
is it only lowercase letters that are allowed in filename?

Reply:
ha nice find, that's been wrong since introduction then... I can't imagine uppercase not being supported in the backend/ client and [the mender artifact regex supports it](https://github.com/mendersoftware/mender-artifact/blob/master/awriter/writer.go#L619)

Reply:
ah, nope it was case independent already, but still good to have it aligned with Mender Artifact 👍